agents/utils/define_crews.py

```
@@ -0,0 +1,52 @@
+import logging
+
+from crewai import Agent, Crew, Task
```

Ruff (0.8.2)

`crewai.Crew` imported but unused

Remove unused import: `crewai.Crew`

(F401)